exporter.go

```diff
@@ -402,7 +414,20 @@ func main() {
 	}
 
 	log.Printf("NGINX Prometheus Exporter has successfully started")
```

Comment:
previously, `log.Printf("NGINX Prometheus Exporter has successfully started")` was called after all cli argument validation happened. 

now it is called before the validation for the case of web.secured-metrics. 

Could you possibly print that log line after you checked that the cert and key file exist?

Reply:
Done, I left different traces to make it easier to detect in log file how it started (https/http)